modules/openapi-generator/src/main/resources/dart-jaguar/class.mustache

```diff
@@ -2,12 +2,12 @@ part '{{classFilename}}.jser.dart';
 
 class {{classname}} {
   {{#vars}}{{#description}} /* {{{description}}} */{{/description}}
-  @Alias('{{{baseName}}}')
+  @Alias('{{baseName}}')
```

Comment:
Minor suggestion: using `{{{baseName}}}` to avoid escaping HTML special characters.

Reply:
Sure :) my bad ^^